### ISSUE TITLE
Tutorial

### ISSUE THREAD

**Issue by Dominiggas** [danielkrupinski/OneByteRadar#2]
Hello, could you do like a tutorial or add an explanation in the readme on how to know what to do there to code that?

**Comment by ranoke** [comment]
Yeah, i would like to see some tutorials about reverse engineering :)
